api/message_test.go

```diff
@@ -366,18 +366,36 @@ func (s *MessageSuite) Test_CreateMessage_failWhenNoMessage() {
 	assert.False(s.T(), s.notified)
```

Comment:
The method `Test_CreateMessage_onlyRequired` in this file should be renamed to `Test_CreateMessage_WithTitle`